src/solvers/twostep_time_solver.C

```diff
@@ -120,14 +123,16 @@ void TwostepTimeSolver::solve()
       old_deltat = _system.deltat;
       _system.deltat *= 0.5;
 
+      _system.update();
```

Comment:
Neither of these are redundant?  Really?

Reply:
They both probably are, sorry this is a leftover from a debugging hack.

Reply:
Fix commit: